contents/tutorials/event-tracking-guide.md

```diff
@@ -3,70 +3,69 @@ title: Complete guide to event tracking
 sidebar: Docs
 showTitle: true
 author: ['ian-vanagas', 'yakko-majuri']
-date: 2022-09-23
+date: 2024-06-11
 featuredVideo: https://www.youtube-nocookie.com/embed/LIJ_TuyMq74
 tags: ['actions', 'events', 'product os', 'product analytics']
 ---
 
-- **Level:** Medium 🦔🦔
-- **Estimated reading time:** 12 minutes ☕️☕️
+Event tracking is the first step in improving your product (after building it). It is the process of capturing user interaction and behavioral data to understand product usage and make improvements. Beyond usage, event tracking is used by growth teams to do experimentation, marketers to track attribution, engineers to understand performance, and more.  
```

Comment:
after building it doesnt quite read right after "first step in improving". Also, I dont think we need to overexplain it
```suggestion
Event tracking is the first step in improving your product. It enables you to understand how users are interacting with your app and how to best grow it.
```